final_prep/mod_zero_hero.rb

```diff
@@ -48,24 +79,71 @@
 #assess_situation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
 #assess_situation(3, announcement, excuse) > should print - "Meh. Hard pass."
 
+test_case_1 = assess_situation(99, "Never fear, the Courageous Curly Bracket is here!", "I think I forgot to lock up my 1992 Toyota Coralla. Be right back.")
+test_case_2 = assess_situation(21, "Never fear, the Courageous Curly Bracket is here!", "I think I forgot to lock up my 1992 Toyota Coralla. Be right back.")
+test_case_3 = assess_situation(3, "Never fear, the Courageous Curly Bracket is here!", "I think I forgot to lock up my 1992 Toyota Coralla. Be right back.")
+
+
 # Declare a new variable - scary_monster - assigned to an hash with the following key/values
 #   - name (string)
 #   - smell (string)
 #   - weight (integer)
 #   - citiesDestroyed (array)
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
-
+scary_monster = {
+  name: "Yard Lurker",
+  smell: "month old grass clippings in your lawn-mower bag during a heat wave",
+  weight: 200,
+  citiesDestroyed: ["SuburbVille", "Pleasant Town"],
+  luckyNumbers: [13, 33, 8],
+  address: {
+    number: 16,
+    street: "Bay Oaks Ct",
+    state: "MO",
+    zip: "63367"
+  }
+}
+
+puts scary_monster.keys
+puts scary_monster.values
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
+
+class SuperHero
+  attr_accessor :name, :super_power, :age, :arch_nemesis, :power_level, :energy_level
+
+  def initialize(name, super_power, age)
+    @name = name
+    @super_power = super_power
+    @age = age
+    @@arch_nemesis = "The Syntax Error"
```

Comment:
The `@@` is declaring a class method which is a place we don't want to go in Mod 0... maybe not even in Mod 1. For now, I'd use the single `@`.